data_steward/cdr_cleaner/cleaning_rules/remove_ehr_data_without_consent.py

```diff
@@ -227,12 +249,23 @@ def get_sandbox_tablenames(self):
     if ARGS.list_queries:
         clean_engine.add_console_logging()
         query_list = clean_engine.get_query_list(
-            ARGS.project_id, ARGS.dataset_id, ARGS.sandbox_dataset_id,
-            [(RemoveEhrDataWithoutConsent,)])
+            ARGS.project_id,
```

Comment:
@MichaelSchmidt82 The chunk of code that adds the arguments to argument parser in this cleaning rule is missing.

Reply:
@MichaelSchmidt82 

`parser.add_argument('--ehr_duplicates_dataset',
                        action='store',
                        dest='ehr_duplicates_dataset',
                        required=True,
                        help='The dataset that includes duplicate records')
    parser.add_argument(
        '--ehr_duplicates_table',
        action='store',
        dest='ehr_duplicates_table',
        required=True,
        help='The table (from the dataset) that includes duplicates PIDs')`

this should be added here as well, if not if you have to run the CR by itself it would fail as we will be sending the unknown arguments